__init__.py

```diff
@@ -171,6 +171,12 @@ def decompile(self, inputfile):
             os.unlink('{}.c'.format(inputfile))
             os.unlink('{}.c.frontend.dsm'.format(inputfile))
 
+            try:
+                conf.close()
+                os.unlink(conf.name)
+            except OSError:
+                pass
+
```

Comment:
We should probably let the `with` clause do the `close()` and do the `os.unlink` on the upper level.

Reply:
I *feel* like there was a reason that I did it specifically like this but I can't see an obvious good reason for it looking at it now. I'll try this on an actual affected installation and see if it works just having the context manager close it.

Reply:
Immediately after writing this, I remembered it was because I needed `conf` in scope for `conf.name`, but it's cleaner just to capture the name inside the `with` and close after.